src/core/frames/frame_controller.ts

```diff
@@ -152,13 +152,16 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
   }
 
   requestFinished(request: FetchRequest) {
-    this.element.removeAttribute("busy")
+    this.element.removeAttribute("aria-busy")
   }
 
   // Form submission delegate
 
   formSubmissionStarted(formSubmission: FormSubmission) {
-
+    const frame = this.findFrameElement(formSubmission.formElement)
+    if (frame) {
+      frame.setAttribute("aria-busy", "")
```

Comment:
Should the value be `"true"` here like above? https://github.com/hotwired/turbo/blob/0c2bd1446fb202eed2a234ba1ccc7250bedba950/src/core/frames/frame_controller.ts#L131-L132

Reply:
Yes! Thank you